modules/dbparser/groupingby.c

```diff
@@ -254,6 +254,30 @@ grouping_by_generate_synthetic_msg(GroupingBy *self, CorrellationContext *contex
   return msg;
 }
 
+static LogMessage *
+grp_sort_remove_context(GroupingBy *self, CorrellationContext *context)
```

Comment:
another minor one:
why is the function name `grp_` ?

Reply:
I got lazy, but not lazy enough to straight remove the prefix. Now I choose a different name, I think it is better.

Reply:
Thanks!
The new name `grouping_by_update_context_and_generate_msg`  is much better.